api/api_test.go

```diff
@@ -2,20 +2,29 @@ package api_test
 
 import (
 	"context"
+	"strings"
 	"testing"
 
 	"github.com/ONSdigital/dp-search-reindex-api/api"
 	"github.com/ONSdigital/dp-search-reindex-api/api/mock"
+	"github.com/ONSdigital/dp-search-reindex-api/config"
 	"github.com/gorilla/mux"
 	. "github.com/smartystreets/goconvey/convey"
 	"net/http/httptest"
 )
 
 func TestSetup(t *testing.T) {
+	cfg, _ := config.Get()
```

Comment:
Sorry didn't spot this before, but there's really never a good reason to skip checking an error. Can only cause problems down hassle down the line.